lib/ansible/modules/cloud/amazon/ec2_elb_facts.py

```diff
@@ -80,6 +80,9 @@
 
 import traceback
 
+from ansible.module_utils.basic import *
```

Comment:
I fixed them in https://github.com/ansible/ansible/pull/22415 which I can fix up once this is merged in.